src/modules/create/game-editor/editors/MapEditor.tsx

```diff
@@ -0,0 +1,47 @@
+
+import React from "react";
+import { EditorProps } from "./EditorType";
+import { styled } from "@mui/material/styles";
+import { useProject } from "src/providers/ProjectProvider";
+import { MapManager } from "@utils/MapManager";
+
+const MapEditorContainer = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  height: "100%",
+  borderRadius: theme.spacing(1),
+  borderTopLeftRadius: 0,
+  backgroundColor: theme.palette.blue[800],
+  gap: theme.spacing(1),
+  boxSizing: "border-box",
+  padding: theme.spacing(1),
+}));
+
+const Top = styled("div")(({ theme }) => ({
```

Comment:
Why do we have to do this? Isn't using `padding` in `MapEditorContainer` enough?